src/main/java/com/couchbase/lite/support/CouchbaseLiteHttpClientFactory.java

```diff
@@ -36,13 +36,15 @@
     private ClearableCookieJar cookieJar;
     private SSLSocketFactory sslSocketFactory;
 
+    private boolean followRedirects = true;
+
     // deprecated
     public static int DEFAULT_SO_TIMEOUT_SECONDS = 60 * 5;
 
     // OkHttp Default Timeout is 10 sec for all timeout settings
-    public static int DEFAULT_CONNECTION_TIMEOUT_SECONDS = 10;
+    public static int DEFAULT_CONNECTION_TIMEOUT_SECONDS = 30;
     public static int DEFAULT_READ_TIMEOUT = DEFAULT_SO_TIMEOUT_SECONDS;
-    public static int DEFAULT_WRITE_TIMEOUT = 10;
+    public static int DEFAULT_WRITE_TIMEOUT = 30;
```

Comment:
Thank you for catching this. I temporary increased for debugging purpose. But I forgot....